### DISCUSSION TITLE
Sparse noise luminance control

### DISCUSSION THREAD

**Discussion by mcarleto** [bonsai-rx/bonsai#1016] [Q&A]
Hi! I have a psychophysics experiment that utilizes a gabor contrast over specified noise. Everything works great, but when using sparse noise to create my background noise, I was wondering if there was any way to control the luminance of the activated pixels, since currently they are either totally on or totally off. For this experiment I would need to also preform this at low luminance levels and need control of the max pixel value. 

Thanks!

Maya 

I've attached a zip of my work flow and input file. 
[Gaborcontrasthuman.bonsai.zip](https://github.com/bonsai-rx/bonsai/files/9446394/Gaborcontrasthuman.bonsai.zip)

**Comment by bruno-f-cruz** [comment]
A `ConvertScale` before updating the texture should allow you to control the contrast.
Hope it helps!

**Reply by mcarleto** [reply]
That did just the trick! Thank you! As a follow up question, I assumed then that I could alter the image in many ways by putting something in front of the update texture, specifically changing the color of the white pixels to be entirely in the green domain. I tried using the colorbalance node and concating with the convert scale. The convert scale still works to lower the luminance but the color remains white. Any ideas?
